presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -841,12 +841,15 @@ public void testCreateUnpartitionedTableAsSelect()
         String tmpTableName = generateRandomTableName();
         // Clean up if temporary table already exists.
         dropTableIfExists(tmpTableName);
-        try {
-            getQueryRunner().execute(session, String.format("CREATE TABLE %s AS SELECT * FROM nation", tmpTableName));
-            assertQuery(String.format("SELECT * FROM %s", tmpTableName), "SELECT * FROM nation");
-        }
-        finally {
-            dropTableIfExists(tmpTableName);
+        String[] tableFormats = {"DWRF", "PARQUET"};
```

Comment:
Can we add to test the format that we won't support? Thanks!

Reply:
done

Reply:
However, it needed extending the presto protocol.